src/ansys/pre_commit_hooks/add_license_headers.py

```diff
@@ -279,6 +283,53 @@ def check_exists(changed_headers, parser, values, proj, missing_headers, i):
     return changed_headers
 
 
+def add_hook_changes(before_hook, after_hook):
```

Comment:
Let's start getting used to adding type hints whenever we can. This will ease things in development.